### PR TITLE
Static pods or kni networking infra

### PR DESCRIPTION
**Pull request by celebdor**
This PR intends to move all the podman+systemd net infra services to static pods

### CODE REVIEW

File: assets/files/etc/kubernetes/manifests/coredns.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    hostPath:
+      path: "/bin/oc"
+  - name: clusterinfo
+    hostPath:
+      path: "/usr/local/bin/clusterinfo"
+  - name: conf-dir
+    empty-dir: {}
+  initContainers:
+  - name: render-corefile
+    image: quay.io/openshift/origin-node:latest
```
Comment (vrutkovs) [inline assets/files/etc/kubernetes/manifests/coredns.yaml]:
```suggestion
image: registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```
This has /usr/libexec/platform-python (also python2) and dig

**Comment by celebdor** [comment]
Will tackle haproxy in a follow-up PR

**Comment by jparrill** [comment]
The PR is working fine for here.

/LGTM

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/535/

**Pull request merged**